### DISCUSSION TITLE
API를 얼마나 쏴야하는가~!~!!!

### DISCUSSION THREAD

**Discussion by ddongule** [woowacourse/react-payments#41] [토론장]
사용자가 매번 어떤 행위를 할 때마다 API를 전송해야 할까요? 
State로 저장해야 할 정보들에는 어떤 것들이 있을까요?

**Comment by zereight** [comment]
비용이 큰 request는 캐싱을 해서, 메모리나 storage에 저장을 하고,
업데이트 기준을 정해서 그 때에만 request를 보내면 되지 않을까요.

state로 들고 있어야하는 정보는 react에서 실시간으로 제어가 가능한, 변할 수 있는 값을 저장해야 되지 않을까여

**Reply by ddongule** [reply]
댑악사건

**Comment by Tanney-102** [comment]
제목만 보고 그루밍인지 알겠어요

**Reply by yujo11** [reply]
빵야빵야 빵야 ~~
